scala/src/main/scala/com/scalableminds/webknossos/wrap/WKWFile.scala

```diff
@@ -165,7 +165,6 @@ class WKWFile(val header: WKWHeader, fileMode: FileMode.Value, underlyingFile: R
 
   def readBlock(x: Int, y: Int, z: Int): Box[Array[Byte]] = {
     for {
-      _ <- (!underlyingFile.isClosed) ?~! error("File is already closed")
```

Comment:
This check is no longer needed because the API `get` calls reopen closed files (only the unsafe memory copy we used before didn’t)